2024-03-04 16:16:12,102 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-04 16:16:12,105 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Configure stats pid to 20520
2024-03-04 16:16:12,105 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-04 16:16:12,105 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\settings
2024-03-04 16:16:12,106 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-04 16:16:12,106 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-04 16:16:12,106 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-04 16:16:12,108 INFO    MainThread:20520 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240304_161612-nqwphfx1\logs\debug.log
2024-03-04 16:16:12,110 INFO    MainThread:20520 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240304_161612-nqwphfx1\logs\debug-internal.log
2024-03-04 16:16:12,110 INFO    MainThread:20520 [wandb_init.py:init():566] calling init triggers
2024-03-04 16:16:12,110 INFO    MainThread:20520 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-04 16:16:12,110 INFO    MainThread:20520 [wandb_init.py:init():591] re-initializing run, found existing run on stack: uiuqqcpr
2024-03-04 16:16:12,113 INFO    MainThread:20520 [wandb_run.py:_finish():1970] finishing run yanzhuo/word2vec_project/uiuqqcpr
2024-03-04 16:16:12,114 INFO    MainThread:20520 [jupyter.py:save_history():445] not saving jupyter history
2024-03-04 16:16:12,114 INFO    MainThread:20520 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-04 16:16:12,114 INFO    MainThread:20520 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-04 16:16:12,115 INFO    MainThread:20520 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-04 16:16:12,117 INFO    MainThread:20520 [wandb_run.py:_restore():2193] restore
2024-03-04 16:16:12,117 INFO    MainThread:20520 [wandb_run.py:_restore():2199] restore done
2024-03-04 16:16:21,042 INFO    MainThread:20520 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-04 16:16:21,044 INFO    MainThread:20520 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-04 16:16:21,069 INFO    MainThread:20520 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-04 16:16:21,445 INFO    MainThread:20520 [wandb_init.py:init():616] starting backend
2024-03-04 16:16:21,446 INFO    MainThread:20520 [wandb_init.py:init():620] setting up manager
2024-03-04 16:16:21,488 INFO    MainThread:20520 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-04 16:16:21,492 INFO    MainThread:20520 [wandb_init.py:init():628] backend started and connected
2024-03-04 16:16:21,571 INFO    MainThread:20520 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-04 16:16:21,584 INFO    MainThread:20520 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'charmap' codec can't decode byte 0x8d in position 23498: character maps to <undefined>
2024-03-04 16:16:21,586 INFO    MainThread:20520 [wandb_init.py:init():720] updated telemetry
2024-03-04 16:16:21,692 INFO    MainThread:20520 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-04 16:16:22,607 INFO    MainThread:20520 [wandb_run.py:_on_init():2262] communicating current version
2024-03-04 16:16:22,840 INFO    MainThread:20520 [wandb_run.py:_on_init():2271] got version response 
2024-03-04 16:16:22,840 INFO    MainThread:20520 [wandb_init.py:init():804] starting run threads in backend
2024-03-04 16:16:23,145 INFO    MainThread:20520 [wandb_run.py:_console_start():2241] atexit reg
2024-03-04 16:16:23,146 INFO    MainThread:20520 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-04 16:16:23,146 INFO    MainThread:20520 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-04 16:16:23,150 INFO    MainThread:20520 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-04 16:16:23,155 INFO    MainThread:20520 [wandb_init.py:init():847] run started, returning control to user process
2024-03-04 16:16:23,155 INFO    MainThread:20520 [wandb_watch.py:watch():51] Watching
2024-03-04 16:17:01,136 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-04 16:17:01,139 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Configure stats pid to 20520
2024-03-04 16:17:01,140 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-04 16:17:01,140 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\settings
2024-03-04 16:17:01,141 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-04 16:17:01,141 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-04 16:17:01,141 INFO    MainThread:20520 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-04 16:17:01,144 INFO    MainThread:20520 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240304_161701-eejo58w9\logs\debug.log
2024-03-04 16:17:01,146 INFO    MainThread:20520 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240304_161701-eejo58w9\logs\debug-internal.log
2024-03-04 16:17:01,146 INFO    MainThread:20520 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000241291A4AA0>
2024-03-04 16:17:01,146 INFO    MainThread:20520 [wandb_init.py:init():566] calling init triggers
2024-03-04 16:17:01,146 INFO    MainThread:20520 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-04 16:17:01,147 INFO    MainThread:20520 [wandb_init.py:init():591] re-initializing run, found existing run on stack: nqwphfx1
2024-03-04 16:17:01,149 INFO    MainThread:20520 [wandb_run.py:_finish():1970] finishing run yanzhuo/word2vec_project/nqwphfx1
2024-03-04 16:17:01,149 INFO    MainThread:20520 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-04 16:17:01,152 INFO    MainThread:20520 [wandb_run.py:_restore():2193] restore
2024-03-04 16:17:01,153 INFO    MainThread:20520 [wandb_run.py:_restore():2199] restore done
2024-03-04 16:17:08,490 INFO    MainThread:20520 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-04 16:17:08,492 INFO    MainThread:20520 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-04 16:17:08,519 INFO    MainThread:20520 [wandb_run.py:_footer_sync_info():3825] logging synced files
